okio/src/test/java/okio/ReadUtf8LineTest.java

```diff
@@ -94,6 +94,20 @@
     }
   }
 
+  @Test public void readUtf8LineStrictWithLimits() throws IOException {
+    data.writeUtf8("ab\nde\n");
+    assertEquals("ab", source.readUtf8LineStrict(4));
+    assertEquals("de", source.readUtf8LineStrict(4));
+
+    data.writeUtf8("abcdef\n");
+    try {
+      source.readUtf8LineStrict(4);
+      fail();
+    } catch (EOFException expected) {
+      assertEquals("\\n not found: size=4 content=6162636465660a…", expected.getMessage());
```

Comment:
It does look odd, now that you mention it. One option: "\n not found in first n bytes of content=6162636465660a…", where n is replaced with the number of bytes scanned (in this case 4).